src/nav-soknad/faktum/TextareaEnhanced.tsx

```diff
@@ -20,7 +20,7 @@ interface OwnProps {
     property?: string;
     faktumId?: number;
     getName?: () => string;
-    getFeil?: (intl: IntlShape) => Feil; // Fjern
+    getFeil?: (intl: IntlShape) => string; // Fjern
     onChange?: (event: any) => any;
     onBlur?: () => void;
     feil?: any; // Type??
```

Comment:
Det skal egentlig være:
 
```
   feil?: Valideringsfeil[]
```

Reply:
Hvis denne uansett skal fjernes lar jeg den stå nå. Og hva slags type er `Valideringsfeil`? Ut fra det jeg kan se i dokumentasjonen til designsystemet, så er feilmeldinger for komponentene av typen `string`.